### ISSUE TITLE
Add "Mute account" in pop-up menu

### ISSUE THREAD

**Issue by Shohreh** [bluesky-social/social-app#6964]
### Describe the Feature

I didn't find [this suggestion](is:open label:feature-request mute account) in the list.

When right-clicking on an account in a thread, it shows "Block account", but no "Mute account".

Cheers,

![image](https://github.com/user-attachments/assets/269d92bf-e411-4bb7-8e45-df82ad7e6d8e)

**Comment by Shohreh** [comment]
No. It's not listed in a _thread_, forcing the user to click more to mute the user by going through their profile.

If the menu has a "Block account", it should logically also have a "Mute account".

**Comment by surfdude29** [comment]
Yes, that's what the issue I linked is about? 🤔
